apps/src/templates/rubrics/StudentSelector.jsx

```diff
@@ -84,9 +84,16 @@ function StudentSelector({
           label: (
             <div className={style.studentDropdownOptionContainer}>
               <div className={style.studentDropdownOption}>
-                <BodyThreeText className={style.submitStatusText}>{`${
-                  student.name
-                } ${student.familyName || ''}`}</BodyThreeText>
+                <BodyThreeText className={style.submitStatusText}>
+                  {student.familyName
+                    ? student.familyName.length < 10
+                      ? `${student.name} ${student.familyName}`
+                      : `${student.name} ${student.familyName.substring(
```

Comment:
Are we missing a case here where the first name is really long and it might get cut off before it gets to the last name?  Maybe an extreme edge case?